torchvision/datasets/image.py

```diff
@@ -0,0 +1,46 @@
+import torch.utils.data as data
+from PIL import Image
+import os.path
+
+def make_dataset(file_name):
+    images = []
+    with open(file_name) as f:
+        for line in f.read().splitlines():
+            img, c = line.split()
+            images.append((img, int(c)))
+    return images
+
+
+def default_loader(path):
+    return Image.open(path).convert('RGB')
+
+
+class ImageData(data.Dataset):
+    """
+    Caffe style ImageData
+
+    Arguments:
+        root: data directory
+        list_file: a file name. Each row of the file: image_path class
+    """
+    def __init__(self, root, list_file, transform=None, target_transform=None,
```

Comment:
it would be nice to allow for a default root of `""`, in case your input file contains absolute paths